pkg/transport/listen_generic.go

```diff
@@ -0,0 +1,13 @@
+//go:build !darwin && !linux && !windows
+// +build !darwin,!linux,!windows
+
+package transport
+
+import (
+	"net"
+	"net/url"
+)
+
+func listenURL(url *url.URL) (net.Listener, error) {
+	return defaultListenURL(url)
+}
```

Comment:
@cfergeau so this is implemented to build it freebsd?

Reply:
On platforms which are not linux not darwin and not windows. The initial request was for BSD-like systems https://github.com/containers/gvisor-tap-vsock/pull/259